### PR TITLE
Create comments on posts

### PR DESCRIPTION
**Pull request by xkb**
@derekdowling have a look :dancer:

### CODE REVIEW

File: app/scripts/components/content/content.jsx [hunk 1]
```diff
@@ -23,10 +25,13 @@ var Content = React.createClass({
   render: function() {
     var content, comments, title;
 
-    if (_.isUndefined(this.props.data)) {
-      return (<i className="fa fa-refresh fa-spin fa-5x"></i>);
+    if (_.isNull(this.props.data)) {
```
Comment (derekdowling) [inline app/scripts/components/content/content.jsx:28]: Not to distract you, but we could probably write the spinner as a mixin that does this. Maybe not.

File: app/scripts/stores/author.js [hunk 2]
```diff
@@ -188,12 +191,36 @@ export default Reflux.createStore({
...
     var post = new Post(postData);
-    post.author = new Author(post.author);
+
+    // Cyclic reference
+    post.author = this.currentAuthor;
+    // add new post
     this.currentAuthor.posts.push(post);
-    this.trigger({displayAuthor: this.currentAuthor});
+    // trigger update
+    this.trigger({displayPosts: this.currentAuthor.sortedPosts()});
```
Comment (derekdowling) [inline app/scripts/stores/author.js:200]: We should probably enforce the sorting in the backend. Unfortunately we still need to sort out the date serializer stuff first.

**Comment by derekdowling** [comment]
Looks good :+1:

File: app/scripts/components/content/content.jsx (hunk 1 above)
Comment (xkb) [inline app/scripts/components/content/content.jsx:28]: Maybe. I'll make an extra issue for it.

**Pull request merged**